pahfit/scripts/run_pahfit.py

```diff
@@ -137,18 +137,19 @@ def main():
     mpl.rc("font", **font)
     mpl.rc("lines", linewidth=2)
     mpl.rc("axes", linewidth=2)
-    mpl.rc("xtick.major", width=2)
-    mpl.rc("ytick.major", width=2)
+    mpl.rc("xtick.major", size=5, width=1)
+    mpl.rc("ytick.major", size=5, width=1)
+    mpl.rc("xtick.minor", size=3, width=1)
+    mpl.rc("ytick.minor", size=3, width=1)
 
-    fig, ax = plt.subplots(figsize=(15, 10))
+    fig, axs = plt.subplots(ncols=1, nrows=2, figsize=(15, 10),
+                            gridspec_kw={'height_ratios': [3, 1]},
+                            sharex=True)
 
-    pmodel.plot(ax, obs_x, obs_y, obs_fit)
-
-    ax.set_yscale("linear")
-    ax.set_xscale("log")
+    pmodel.plot(axs, obs_x, obs_y, obs_unc.value, obs_fit)
 
     # use the whitespace better
-    fig.tight_layout()
+    fig.subplots_adjust(hspace=0)
```

Comment:
Why the change from tight_layout to subplots_adjust?  Not a problem, I'm curious and want to learn.

Reply:
I had fig.subplots_adjust(hspace=0) above the tight_layout line without removing the tiight_layout() but then tight_layout overrode hspace=0 and made the panels with an offset.